packages/nouns-contracts/contracts/libs/InMemoryMapping.sol

```diff
@@ -0,0 +1,84 @@
+// SPDX-License-Identifier: GPL-3.0
+
+/*********************************
+ * ░░░░░░░░░░░░░░░░░░░░░░░░░░░░░ *
+ * ░░░░░░░░░░░░░░░░░░░░░░░░░░░░░ *
+ * ░░░░░░█████████░░█████████░░░ *
+ * ░░░░░░██░░░████░░██░░░████░░░ *
+ * ░░██████░░░████████░░░████░░░ *
+ * ░░██░░██░░░████░░██░░░████░░░ *
+ * ░░██░░██░░░████░░██░░░████░░░ *
+ * ░░░░░░█████████░░█████████░░░ *
+ * ░░░░░░░░░░░░░░░░░░░░░░░░░░░░░ *
+ * ░░░░░░░░░░░░░░░░░░░░░░░░░░░░░ *
+ *********************************/
+
+pragma solidity ^0.8.19;
+
+/**
+ * @notice A library for short lived in memory mappings.
+ * This only works when the keys, in this case clientId, are of limited size so that they we can create
+ * an array in memory for all the possible values
+ */
+library InMemoryMapping {
```

Comment:
the name of this lib gives the feeling that it's general purpose, when in fact the code is pretty specific for clientIds and balances, so I'm thinking giving it a more specific name is good.

wdyt? @davidbrai

Reply:
@eladmallel naming is hard. do you have an idea for a name?

Reply:
Maybe `ClientRewardsMemoryMapping`?

Reply:
done